### ISSUE TITLE
Cypress plugin doesn't follow paths

### ISSUE THREAD

**Issue by stephenwade** [webpro/knip#558]
I'm trying to set up Knip in an existing project. I discovered that the Cypress plugin doesn't follow `paths` as inferred from `tsconfig.json`.

My `cypress.config.ts` files import some common functions from a lib that is accessed via a path. When I run Knip, I get the following error:

```
$ npm run knip

> knip-issue-repro-stunning-happiness@0.0.1 knip
> knip

Analyzing workspace ....
Error loading /Users/stephen.wade/git/knip-issue-repro-stunning-happiness/cypress.config.ts
Reason: Cannot find module '@/cypress'
Require stack:
- /Users/stephen.wade/git/knip-issue-repro-stunning-happiness/cypress.config.ts
```

If I rename `cypress.config.ts` to `main.ts`, Knip says there are no issues, demonstrating that Knip can get the `paths` properly in that case.

Instructions to reproduce:

1. Clone the minimal reproduction: https://github.com/stephenwade/knip-issue-repro-stunning-happiness
2. `npm ci`
3. `npm run knip` – The error will happen.
4. `mv cypress.config.ts main.ts && npm run knip` – The error will not happen.

**Comment by webpro** [comment]
The issue is probably that the `compilerOptions.paths` are not loaded/used during script execution of `cypress.config.ts` (this uses jiti for script loading).

You can try the same using Bun: `bunx --bun knip` (this uses bun for script loading).

**Comment by webpro** [comment]
Thanks for checking, I need to document this (e.g. using TS aliases in dynamic Jest configs has the same issue I believe)

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.1.5. See [Release 5.1.5](https://github.com/webpro/knip/releases/tag/5.1.5) for release notes._
